### DISCUSSION TITLE
FIS fetching BAP public API key from LookUp API

### DISCUSSION THREAD

**Discussion by nemishzalavadiya-pirimid** [ONDC-Official/ONDC-FIS-Specifications#51] [Q&A]
To get public key of buyer app to do `look up` required request is,
```
{
  "subscriber_id": "string",
  "country": "string",
  "ukId": "string",
  "city": "std:080",
  "domain": "nic2004:52110",
  "type": "BAP or BPP"
}
```
This all fields are `mandatory` for `look up api according` to swagger doc: 
[Onboarding Swagger Doc Link](https://app.swaggerhub.com/apis-docs/ONDC/ONDC-Registry-Onboarding/2.0.5#/ONDC%20Network%20Participant%20Onboarding/post_lookup)

But we do not have this mandatory in `Core API` for FIS API collection in this swager doc:
[FIS Core API Swagger](https://ondc-official.github.io/ONDC-FIS-Specifications/#/Consumer%20Platform/post_on_select)
> refer `version` : `draft-loan`, `Context` object in all request.

Please help use how to find `public key` we don't know the `city`? Country `IND` is by default so no issue there.

Eigher make `lookUp` API work only based on `SubId` and `UkId` or make `Context` as whole mandatory in Core FIS API Collection.

@92shreyansh

**Comment by 92shreyansh** [comment]
The current implemnetation of lookup api will function with any 2 parameters

**Reply by nemishzalavadiya-pirimid** [reply]
Can you please update `Mandatory fields` in `lookUp` API to avoid confusion?

**Reply by nemishzalavadiya-pirimid** [reply]
And can you please clarify which 2 parameters?
1. subscriberId & ukId
2. subscriberId & type
3. ukId & type

Mostly `subscriberId & ukId`, please confirm.
